beacon-chain/sync/metrics.go

```diff
@@ -89,6 +89,47 @@ var (
 			Buckets: []float64{250, 500, 1000, 1500, 2000, 4000, 8000, 16000},
 		},
 	)
+
+	// Attestation processing errors and successes.
+	unaggregatedAttsProcessedCount = promauto.NewCounter(prometheus.CounterOpts{
```

Comment:
We probably want to think about this more. There are **tons** of parallel attestations coming in on per slot, both aggregated and unaggregated. We don't want such instrumentation to cause extra loads to the node

Reply:
it is cheap to increase a single counter op, I believe, and the node can already handle a lot without issues here. It's never been a problem in our infra